src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -0,0 +1,122 @@
+import _ from "lodash"
+import { useQuery, UseQueryResult } from "react-query"
+import type { StringKeyOf } from "type-fest"
+
+import { settingsService } from "services/SettingsService"
+
+import {
+  BackendSiteSettings,
+  SiteColourSettings,
+  SiteSettings,
+  SiteSocialMediaSettings,
+} from "types/settings"
+
+import { SETTINGS_CONTENT_KEY } from "../queryKeys"
+
+import { BE_TO_FE } from "./constants"
+
+const DEFAULT_BE_STATE = {
+  title: "",
+  description: "",
+  favicon: "",
+  shareicon: "",
+  facebook_pixel: "",
+  google_analytics: "",
+  linkedin_insights: "",
+  is_government: "",
+  contact_us: "",
+  feedback: "",
+  faq: "",
+  show_reach: "",
```

Comment:
I think `is_government` and `show_reach` may need to default to `false`, otherwise the value being passed back to the BE might be an empty string

Reply:
good catch, fixed!